docs/learn/learn-async-backing.md

```diff
@@ -7,104 +7,312 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
+
+The diagram below shows the pipelining table for synchronous backing.
+
+![sync-backing-pipeline](../assets/sync-backing-pipeline.png)
+
+Backing (B) and inclusion (I) of each parablock happen on different relay chain blocks, and new
+parablocks are generated once the previous parablock has been included.
+[Contextual execution](#contextual-execution) is driven by the latest included parablock in the most
+recent relay parent. After 24 seconds, two parablocks, P1 and P2, have been included in the relay
+chain, and P3 has been backed.
+
+In synchronous backing we can imagine parablocks on a conveyor belt. The belt is synched with the
+relay chain: It takes six seconds (one relay block) to generate and back a parablock. It then takes
+another six seconds to make that parablock available and mark it as included in a relay block. The
+full process, from one end of the belt to the other, takes twelve seconds, and cannot be started for
+parablock N + 1 until it is fully complete for parablock N (no [pipelining](#pipelining)).
+
+The diagram below shows parablocks on the conveyor belt, on their way from being generated to being
+backed and included into the relay parent.
+
+![sync-backing-nofill](../assets/sync-backing-nofill.png)
+![sync-backing-nofill-legend](../assets/sync-backing-nofill-legend.png)
+
+Parablock 1 (P1) is included in the latest relay chain parent 1 (R1) after 6 seconds. Once P1 is
+included, Parablock 2 (P2) can be generated using the included P1 in R1 as execution context. The
+next parablock P3 can be generated after P2 is backed into R2 and included into R3, i.e. after 12
+seconds.
 
-This pipeline will allow [collators](./learn-parachains-protocol.md#collators) to include an
-estimated ~3-5x more transactions/data while speeding up parachain block times from 12 to 6 seconds.
-In short, Polkadot with asynchronous backing will deliver an estimated ~6-10x more blockspace to its
-parachains.
+The diagram below combines the previous diagrams and adds some context about blockspace.
+
+![sync-backing-fill](../assets/sync-backing-fill.png)
+![sync-backing-fill-legend](../assets/sync-backing-fill-legend.png)
+
+Because P2 is rushing to be backed into R2 in 6 seconds, there are less than 6 seconds (~ 0.5
+seconds) to fill it, since the other 5.5 seconds are needed to get the block backed on chain. In
+this scenario, P2 is filled up to 70%. After the inclusion of P2 into R3, Parablock 3 (P3) will be
+generated, filled to 50%, and backed into R4 in 6 seconds. This shows that, if blockspace demand
+decreases, collators will always have 0.5 seconds to fill up blocks, leading to blocks that contain
+less and less data.
+
+The parablock generation and backing are bound together within a six-second window that limits the
+amount of data a collator can add to each parablock. Essentially, a parablock is limited to the
+requirement of being backed in six seconds, leaving little time for its generation and its
+blockspace to be properly filled.
```

Comment:
It's really that collators are waiting to see that the previous block was included before starting a new collation. That's both why they're limited to 12s and why the time they have to generate the block is so short.